templates/settings/server_info.mustache

```diff
@@ -0,0 +1,74 @@
+<!--
+  ~ This file is part of the QuestionPy Moodle plugin - https://questionpy.org
+  ~
+  ~ Moodle is free software: you can redistribute it and/or modify
+  ~ it under the terms of the GNU General Public License as published by
+  ~ the Free Software Foundation, either version 3 of the License, or
+  ~ (at your option) any later version.
+  ~
+  ~ Moodle is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  ~ GNU General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU General Public License
+  ~ along with Moodle.  If not, see <http://www.gnu.org/licenses />.
+-->
+{{!
+    @template qtype_questionpy/settings/server_info
+
+    Displays information about the server and the server status.
+
+    Context variables required for this template:
+    * status - server information and status
+    * description - localised field description
+
+    Example context (json):
+    {
+        "name":"questionpy-server",
+        "version":"0.1.0",
+        "allow_lms_packages":true,
+        "max_package_size":20971520,
+        "usage":{
+            "requests_in_process":0,
+            "requests_in_queue":0
+        }
+    }
+}}
+<div>
+    <h3 class="qpy-server-info-section-title">{{#str}} server_info_title_general, qtype_questionpy {{/str}}</h3>
+    <table class="qpy-server-info-table">
+        <tr>
+            <th class="qpy-server-info-th">{{#str}} server_info_name, qtype_questionpy {{/str}}</th>
+            <td class="qpy-server-info-td">{{ name }}</td>
+        </tr>
+        <tr>
+            <th class="qpy-server-info-th">{{#str}} server_info_version, qtype_questionpy {{/str}}</th>
+            <td class="qpy-server-info-td">{{ version }}</td>
+        </tr>
+        <tr>
+            <th class="qpy-server-info-th">{{#str}} server_info_allow_lms_packages, qtype_questionpy {{/str}}</th>
+            <td class="qpy-server-info-td">
+                {{#allowlmspackages}}Yes{{/allowlmspackages}}
+                {{^allowlmspackages}}No{{/allowlmspackages}}
```

Comment:
Das müsste noch lokalisiert werden. Yes/No sind in der `core` Komponente von Moodle als Übersetzungen schon definiert, d.h. `{{#str}} yes, core {{/str}}` müsste funktionieren.